value_sensors/template.lua

```diff
@@ -28,6 +33,9 @@ function ValueSensor.new(name)
         self.settings = sensor_settings
 
         owner[self.name].caption = self:get_line()
+        if self:get_font_color() ~= nil then
+            owner[self.name].style.font_color = self:get_font_color()
+        end
```

Comment:
See above about simplifying here; it could be as simple as `owner[self.name].style.font_color = self.color`.